tests/test_geoprocessing.py

```diff
@@ -2448,8 +2448,7 @@ def test_transform_box(self):
         # plotting it in a GIS polygon, so the expected result below is
         # regression data
         expected_result = [
-            453189.3366727062, 4918131.085894576,
-            468484.1637522648, 4952660.678869661]
+            453188.671769, 4918131.799327, 468483.727558, 4952661.553935]

```

Comment:
I assume the difference here is because of numerical roundoff or something? It does seem like it's within `1e-5` or so. Maybe rather that putting the exact decimal to round to the expected cutoff accuracy? (I know it wasn't like that before but worth saying what's really expected since you're in there)...

Reply:
Might be related to this update? Seems like a more accurate transform? "Off by a couple of meters" caught my eye. https://trac.osgeo.org/gdal/wiki/rfc73_proj6_wkt2_srsbarn#WGS84Pivot